docs/docs/migration-guide.mdx

```diff
@@ -469,6 +469,44 @@ response_selector_output.get("default")
 
   Alternatively, you can train a Core model without NLU using `rasa train core`.
```

Comment:
this is a separate thing - but we don't talk about `core` anymore, so this docs section should be renamed. I'll make a note for it on the docs to dos